CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - CoreTelephony should not be added to tvOS builds [#67](https://github.com/PostHog/posthog-ios/pull/67)
 - Remote notifications methods do not throw if no default implementation [#67](https://github.com/PostHog/posthog-ios/pull/67)
+- Update device type [#63](https://github.com/PostHog/posthog-ios/pull/63)
```

Comment:
Add as a `## Next` on the top since 2.0.4 is already published.

Eg:

## Next

- Update device type [#63](https://github.com/PostHog/posthog-ios/pull/63)

Reply:
I tried to push the fix into your branch but its protected.

Reply:
Updated. I'll check about branch protection.